src/runtime_src/core/tools/xbutil2/SubCmdAdvanced.cpp

```diff
@@ -46,6 +46,7 @@ SubCmdAdvanced::SubCmdAdvanced(bool _isHidden, bool _isDepricated, bool _isPreli
   setIsPreliminary(_isPreliminary);
 
   m_commonOptions.add_options()
+    ("device,d", boost::program_options::value<decltype(m_device)>(&m_device), "The Bus:Device.Function (e.g., 0000:d8:00.0) device of interest")
```

Comment:
Do we need this device. "xrt-smi advanced --device <>" Is not a valid command. We have devices inside the suboptions

Reply:
Ok, please make sure that all possible option combos work after this change!

Reply:
Only clocks and preemption are used as valida reports right now. Generating preemption exits silently and clocks works fine :
```
Z:\Repos\XRT-MCDM-FORK\XRT-MCDM\build\WRelease\xilinx\xrt>xrt-smi advanced --report preemption

Z:\Repos\XRT-MCDM-FORK\XRT-MCDM\build\WRelease\xilinx\xrt>xrt-smi advanced --report clocks

Clocks
  H Clock                : 800 MHz
  MP-NPU Clock           : 400 MHz
```